.github/workflows/setup_miniconda.sh

```diff
@@ -0,0 +1,7 @@
+# This is a bash INCLUDE file. Source it don't run it.
```

Comment:
maybe `activate_miniconda.sh` is a better name because this doesn't install or configure anything unlike the other `setup_` scripts.